ui/components/component-library/text-field-base/text-field-base.scss

```diff
@@ -40,6 +40,7 @@
   &__input {
     border: none;
     height: 100%;
+    width: 100%;
```

Comment:
This fixes an issue I ran into using the read only input in the icon search page. I've checked other component libraries and it seems they do the same https://chakra-ui.com/docs/components/input/usage

<img width="1440" alt="Screen Shot 2022-11-21 at 10 54 18 PM" src="https://user-images.githubusercontent.com/8112138/203245458-9502915b-60d5-4242-8ad3-25b9bdb865b9.png">